### PR TITLE
Remove Interface static

### PR DESCRIPTION
**Pull request by shargon**
Preparing for dotnet standard 2.1

### CODE REVIEW

File: src/Neo.Cryptography.BLS12_381/INumber.cs [hunk 1]
```diff
@@ -2,36 +2,56 @@ namespace Neo.Cryptography.BLS12_381;
...
-    static abstract T operator *(in T x, in T y);
+    //static abstract T operator -(in T x);
+    //static abstract T operator +(in T x, in T y);
+    //static abstract T operator -(in T x, in T y);
+    //static abstract T operator *(in T x, in T y);
+
+    T Negate();
+    T Sum(in T value);
+    T Subtract(in T value);
+    T Pow(in T value);
```
Comment (cschuchardt88) [inline src/Neo.Cryptography.BLS12_381/INumber.cs]: This should be called `multiply`. Not `Pow`.

**Review by cschuchardt88 — CHANGES_REQUESTED** [review]

File: src/Neo.Cryptography.BLS12_381/INumber.cs (hunk 1 above)
Comment (shargon) [reply src/Neo.Cryptography.BLS12_381/INumber.cs]: You are right

File: src/Neo.Cryptography.BLS12_381/INumber.cs [hunk 2]
```diff
@@ -2,36 +2,56 @@ namespace Neo.Cryptography.BLS12_381;
...
-    static abstract T operator -(in T x);
-    static abstract T operator +(in T x, in T y);
-    static abstract T operator -(in T x, in T y);
-    static abstract T operator *(in T x, in T y);
+    //static abstract T operator -(in T x);
+    //static abstract T operator +(in T x, in T y);
+    //static abstract T operator -(in T x, in T y);
+    //static abstract T operator *(in T x, in T y);
+
+    T Negate();
```
Comment (shargon) [inline src/Neo.Cryptography.BLS12_381/INumber.cs:16]: We can add `T One()`

**Comment by shargon** [comment]
merge?

**Comment by Jim8y** [comment]
@cschuchardt88

**Comment by cschuchardt88** [comment]
@shargon @Jim8y looks good, but maybe check `core` `applicationengine` its used in contracts `syscall`.

**Review by Jim8y — APPROVED** [review]

**Pull request merged**